cmd/cashierd/main.go

```diff
@@ -309,18 +314,82 @@ func certStore(config string) (store.CertStorer, error) {
 	return cstore, err
 }
 
+func dropPrivileges(runAsUser string) error {
+	// Lookup privsep user
+	if runAsUser == "" {
+		runAsUser = "nobody"
+	}
+	usr, err := user.Lookup(runAsUser)
+	if err != nil {
+		log.Fatal(err)
+	}
+	uid, err := strconv.Atoi(usr.Uid)
+	if err != nil {
+		log.Fatal(err)
+	}
+	gid, err := strconv.Atoi(usr.Gid)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Drop privileges
+	err = syscall.Setgid(gid)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = syscall.Setuid(uid)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return err
+}
+
 func main() {
+	// Privileged section
 	flag.Parse()
 	config, err := readConfig(*cfg)
 	if err != nil {
 		log.Fatal(err)
 	}
+
 	fs.Register(config.AWS)
 	signer, err := signer.New(config.SSH)
 	if err != nil {
 		log.Fatal(err)
 	}
 
+	logfile := os.Stderr
+	if config.Server.HTTPLogFile != "" {
+		logfile, err = os.OpenFile(config.Server.HTTPLogFile, os.O_WRONLY|os.O_APPEND|os.O_CREATE, 0640)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	addr := fmt.Sprintf("%s:%d", config.Server.Addr, config.Server.Port)
+	l, err := net.Listen("tcp", addr)
```

Comment:
I think just put the `fmt.Sprintf` in here:
```
l, err := net.Listen("tcp", fmt.Sprintf("%s:%d", config.Server.Addr, config.Server.Port))
```
It's not that long and `addr` isn't used anywhere else.